src/TracedRNumber.jl

```
@@ -231,6 +229,28 @@ function Base.float(x::TracedRNumber{T}) where {T}
     return TracedUtils.promote_to(TracedRNumber{float(T)}, x)
 end
 
+using Reactant: ReactantFloat
+
+Base.round(A::TracedRNumber{<:ReactantFloat}) = Base.round(A, RoundNearest)
+Base.floor(A::TracedRNumber{<:ReactantFloat}) = Base.round(A, RoundDown)
+Base.ceil(A::TracedRNumber{<:ReactantFloat}) = Base.round(A, RoundUp)
+
+function Base.round(A::TracedRNumber{<:ReactantFloat}, ::RoundingMode{R}) where {R}
+    if R == :Nearest
+        Ops.round_nearest_even(A)
+    elseif R == :Up
+        Ops.ceil(A)
+    elseif R == :Down
+        Ops.floor(A)
+    else
+        @error "$R is unsupported"
```

throw an actual error instead of logging it.